api/utils/extapi.go

```diff
@@ -0,0 +1,35 @@
+package utils
```

Comment:
If you're going to keep your extapi service in the utils package it's fine, although I would recommend moving the logic to it's own package

Reply:
If you want to keep this logic here, when decoupling just remember to make a struct dedicated only to the extapi logic, and another one dedicated to the csv logic